core/module/used_refinements_spec.rb

```diff
@@ -0,0 +1,32 @@
+require_relative '../../spec_helper'
+
+describe "Module.used_refinements" do
+  ruby_version_is "3.2" do
+    it "returns list of all refined modules used in current scope" do
+      refined_integer = nil
+      refined_string = nil
+      refined_items = []
+
+      r1 =  Module.new do
+        refine Integer do
+          refined_integer = self
+        end
+      end
+
+      r2 = Module.new do
+        refine String do
+          refined_string = self
+        end
+      end
```

Comment:
**suggestion:** if `r1` and `r2` are called after _refinement_ then it's not fully correct. It's just modules-containers where the refinements are defined, nothing more. So they could be names just `m1` and `m2` for instance.